src/components/visualisation-bar/index.css

```diff
@@ -0,0 +1,150 @@
+  color: var(--visual-disabled-text-color);
+}
+
+.hx-visualisation-bar.error {
+  --progress-color: var(--visual-negative-color);
+}
+
+.hx-visualisation-bar-title,
+.hx-visualisation-bar-count {
+  display: flex;
+}
+
+.hx-visualisation-bar-title {
+  width: 70%;
+}
+
+.hx-visualisation-bar-count {
+  width: 30%;
+  flex-direction: row-reverse;
+}
+
+.hx-visualisation-bar-fill-wrapper {
+  max-height: var(--visual-fill-height);
+  height: var(--visual-fill-height);
+  background-color: var(--visual-text-color);
+  border-radius: var(--visual-border-radius);
+}
+
+.hx-visualisation-bar-fill {
+  --text-color: var(--visual-text-color);
+}
+
+.hx-visualisation-bar-fill.inversed-text-color {
+  --text-color: var(--visual-inversed-text-color);
+}
+
+.hx-visualisation-bar-content {
+  margin-top: 15px;
+}
+
+.hx-visualisation-bar-fill-wrapper,
+.hx-visualisation-bar-label-wrapper {
+  min-width: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
+.hx-visualisation-bar-fill {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  transition: background-color ease 0.25s;
+  color: var(--text-color);
+  font-weight: bold;
+  height: var(--visual-fill-height);
+}
+
+.hx-visualisation-bar-label {
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  transition: width ease 0.25s;
+  font-size: 12px;
+}
+
+.hx-animate .hx-visualisation-bar-p-wrapper {
+  transition: width ease 0.25s;
+}
+
+@media only screen and(min-width: 700px) {
+  .hx-visualisation-bar-p-wrapper {
+    min-width: 50px;
+  }
+}
+
+.hx-visualisation-bar-progress .hx-visualisation-bar-fill {
+  background-color: var(--progress-color);
+  border-top-left-radius: var(--visual-border-radius);
+  border-bottom-left-radius: var(--visual-border-radius);
+}
+
+.hx-visualisation-bar-buffer .hx-visualisation-bar-fill {
+  background-color: var(--visual-buffer-color);
+}
+.hx-visualisation-bar.disabled .hx-visualisation-bar-fill-wrapper {
+  border: 1px solid var(--visual-disabled-text-color);
+}
+
+.hx-visualisation-bar.disabled .hx-visualisation-bar-fill-wrapper > div {
+  display: none;
+}
+
+.hx-visualisation-bar.disabled .hx-visualisation-bar-progress .hx-visualisation-bar-fill,
+.hx-visualisation-bar.disabled .hx-visualisation-bar-buffer .hx-visualisation-bar-fill,
+.hx-visualisation-bar.disabled .hx-visualisation-bar-balance .hx-visualisation-bar-fill {
+  background-color: var(--visual-text-color);
+}
+
+.without-balance.without-buffer .hx-visualisation-bar-progress .hx-visualisation-bar-fill {
+  border-top-right-radius: var(--visual-border-radius);
+  border-bottom-right-radius: var(--visual-border-radius);
+}
+
+.without-progress .hx-visualisation-bar-buffer .hx-visualisation-bar-fill {
+  border-top-left-radius: var(--visual-border-radius);
+  border-bottom-left-radius: var(--visual-border-radius);
+}
+
+.without-balance .hx-visualisation-bar-buffer .hx-visualisation-bar-fill {
```

Comment:
All classes should be prefixed with `hx-` and in the case of component classes with `hx-<component name>` (i.e. `hx-visualization-bar-...`)